api/security.go

```diff
@@ -443,12 +466,37 @@ func GetImageDeployments(c *gin.Context) {
 			}
 		}
 	}
-	var releaseList []string
-	for k := range releaseMap {
-		releaseList = append(releaseList, k)
-	}
 
-	c.JSON(http.StatusOK, releaseList)
+	releases := []pkgHelm.ListDeploymentResponse{}
+	if activeReleases != nil && len(activeReleases.Releases) > 0 {
+		for _, r := range activeReleases.Releases {
+			if ok := releaseMap[r.Name]; ok {
+				createdAt := time.Unix(r.Info.FirstDeployed.Seconds, 0)
+				updated := time.Unix(r.Info.LastDeployed.Seconds, 0)
+				chartName := r.GetChart().GetMetadata().GetName()
+
+				body := pkgHelm.ListDeploymentResponse{
```

Comment:
This code snippet seems to be identical with `api/ListDeployments` thus should be factored out into common method and used from `api/ListDeployments` and `GetImageDeployments`.